Readability.js

```diff
@@ -867,6 +878,41 @@ Readability.prototype = {
           lastScore = parentOfTopCandidate.readability.contentScore;
           parentOfTopCandidate = parentOfTopCandidate.parentNode;
         }
```

Comment:
I replied this [here][here].

[here]: https://github.com/mozilla/readability/pull/309/files?diff=unified#r95936960